src/commands/synthetics/README.md

```diff
@@ -145,18 +145,25 @@ const { results, summary } = await synthetics.executeTests(...)
 
 ### Use a proxy
 
-It is possible to configure a proxy to be used for outgoing connections to Datadog using the `proxy` key of the global configuration file.
+You can configure a proxy to be used for outgoing connections to Datadog. To do this, use the `proxy` key of the global configuration file or the `HTTPS_PROXY` environment variable.
+
+⚠️**Note**: This is the only exception where the the global configuration file takes precedence over the environment variable. There is no option to set this through the CLI.
```

Comment:
I'd say, let's remove the ⚠️ character for now, I'm not sure how it'll render in the documentation page.
Also, removing a double `the`.
```suggestion
**Note**: This is the only exception where the global configuration file takes precedence over the environment variable. There is no option to set this through the CLI.
```